runtime/connectors/local-file/local-file_test.go

```diff
@@ -0,0 +1,49 @@
+package local_file_test
+
+import (
+	"context"
+	"github.com/rilldata/rill/runtime/connectors"
+	"github.com/rilldata/rill/runtime/connectors/sources"
+	"github.com/rilldata/rill/runtime/drivers"
+	test_utils "github.com/rilldata/rill/runtime/test-utils"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+var curPath, _ = os.Getwd()
+var TestFilesPath = filepath.Join(curPath, "/../../../web-local/test/data/")
```

Comment:
Can we just use the relative path without the working dir? I believe Go tests are guaranteed to have the working dir set to the test package